src/component/LogPages/LoginPage/LoginPage.scss

```diff
@@ -0,0 +1,57 @@
+.loginPage-text{
+  margin: 65px 0px 5px 0px;
+
+  font-family: Roboto;
+  font-size: 12px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.04px;
+  text-align: center;
+  color: #ff0000;
+}
+
+.loginPage-password{
+  width: 330px;
+  margin-top: 65px;
+  position: relative;
+}
+
+.loginPage-password img{
+  border-bottom: solid 1px #969696;
+  position: absolute;
+  bottom: 0;
+}
+
+.loginPage-password input{
+  border: none;
+  border-bottom: solid 1px #969696;
+  width: 255px;
+  margin-left: 24px;
+
+  font-family: Roboto;
+  font-size: 16px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.05px;
+  color: #969696;
+}
+
+.loginPage-password span{
+  border-bottom: solid 1px #969696;
+  position: absolute;
+  bottom: 0%;
+
+  font-family: Roboto;
+  font-size: 12px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.04px;
+  color: #969696;
+}
+
```

Comment:
img, input and span here could be included in the loginPage-password class when writing scss:
```
.loginPage-password{
  width: 330px;
  margin-top: 65px;
  position: relative;
  
  img{
    border-bottom: solid 1px #969696;
    position: absolute;
    bottom: 0;
     }

  input{
      border: none;
      border-bottom: solid 1px #969696;
      width: 255px;
      margin-left: 24px;
    
      font-family: Roboto;
      font-size: 16px;
      font-weight: normal;
      font-stretch: normal;
      font-style: normal;
      line-height: normal;
      letter-spacing: 0.05px;
      color: #969696;
    }
  span {
    border-bottom: solid 1px #969696;
    position: absolute;
    bottom: 0%;
  
    font-family: Roboto;
    font-size: 12px;
    font-weight: normal;
    font-stretch: normal;
    font-style: normal;
    line-height: normal;
    letter-spacing: 0.04px;
    color: #969696;
  }
}
```